### ISSUE TITLE
[BUG] Constant error log from event_route

### ISSUE THREAD

**Issue by bcnewlin** [OpenSIPS/opensips#1932]
**OpenSIPS version you are running**

```
version: opensips 2.4.6 (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, F_MALLOC, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
git revision: 59400f778
main.c compiled on 19:38:09 Jan  8 2020 with gcc 7
```

**Describe the bug**

OpenSIPS outputs a constant stream of errors:
`Jan  8 19:42:13 [378] ERROR:event_route:event_route_handler: invalid receive sock info`
The output is many thousands of messages per second. The errors begin as soon as OpenSIPS starts; no SIP messages are received.

We have verified this error was introduced in 59400f778b633d43d6d9e7d33fba4658c9c7f4fd.

The errors occur even when no event_routes are defined and no custom events are used. It appears the only requirement is loading the event_route module. The errors were received even with a config file that contained only a single route:

```
route {
    send_reply("503", "Service Unavailable");
}
```

**To Reproduce**

1. Start OpenSIPS with event_route module loaded
2. Inspect log output

**Expected behavior**

The error is not expected.

**Relevant System Logs**

```
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
Jan  8 21:31:12 [357] ERROR:event_route:event_route_handler: invalid receive sock info
```

**OS/environment information**
 - Operating System:  Docker amazonlinux:2
 - OpenSIPS installation:  git
 - other relevant information:

**Additional context**

**Comment by bcnewlin** [comment]
I am attaching the minimal config file I have found to reproduce the error.
[opensips.cfg.txt](https://github.com/OpenSIPS/opensips/files/4037554/opensips.cfg.txt)

**Comment by bcnewlin** [comment]
I have also confirmed the issue does not occur on native Amazon Linux 2. It appears to be restricted to Docker.

**Comment by bcnewlin** [comment]
I've now confirmed the issue exists on Docker using CentOS7. So it appears to be an issue with Docker, not the amazonlinux:2 image specifically.

**Comment by bcnewlin** [comment]
I'm attaching the debug logs from a startup attempt using the config file provided in a previous comment.
[opensips.log](https://github.com/OpenSIPS/opensips/files/4038255/opensips.log)

**Comment by razvancrainea** [comment]
Bug identified, working on a fix now.

**Comment by bcnewlin** [comment]
@razvancrainea Do you need anything else from me? I realized my previous test on linux (no Docker) was not correct so I had not really determined that the problem was only on Docker. I was going to redo that today unless it is not necessary.

**Comment by bcnewlin** [comment]
@razvancrainea I've verified the fix in 8179e1f8d9d580ca3c4d1644cc225bfd0739bd03 does not output the error.

**Comment by razvancrainea** [comment]
Yeah, this ticket should have already been fixed by commit 8179e1f8d9d580ca3c4d1644cc225bfd0739bd03, not sure why it did not pick up the close command from the commit message (Close #1932).
